app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -1057,6 +1068,13 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         poiTapPoint = null
     }
 
+    override fun onSharedPreferenceChanged(preferences: SharedPreferences?, key: String?) {
```

Comment:
Do we really need a preference listener here? By definition `MainActivity` will be in the background whenever this value changes. Would it suffice to simply update the value in `onResume()` or maybe better yet `onPrepareOptionsMenu()`?